docker_xylem/service.py

```diff
@@ -102,12 +112,28 @@ def unmount_volume(self, request, data):
         path = os.path.join(self.mount_path, name)
 
         try:
-            yield self._umount_fs(path)
-            defer.returnValue({"Err": None})
+            if (yield self._umount_fs(path)):
+                defer.returnValue({"Err": None})
+            else:
+                # Try older mount paths
+                paths = self.get_paths(name)
+                for path in paths:
+                    yield self._umount_fs(path)
+
+                defer.returnValue({"Err": None})
```

Comment:
Would it make sense to always try unmount from old paths as well? What if we somehow ended up with the same volume mounted in multiple places?